app/experimenter/kinto/tasks.py

```diff
@@ -120,7 +120,7 @@ def handle_rejection(kinto_client):
     generate_nimbus_changelog(
         experiment,
         get_kinto_user(),
-        message=f'Rejected: {collection_data["last_reviewer_comment"]}',
```

Comment:
Feel free to push back, but rejecting a change from the Nimbus console does _not_ prefix the message with "Rejected:", so this just makes rejections that come from RS more consistent.